src/pages/Withdrawals/index.tsx

```diff
@@ -0,0 +1,744 @@
+    }
+
+    thead th {
+      vertical-align: bottom;
+    }
+
+    tbody + tbody {
+      border-top: 2px solid #dee2e6;
+    }
+  }
+  section.actionable {
+    background-color: white;
+    padding: 2rem;
+    @media only screen and (max-width: ${p => p.theme.screenSizes.medium}) {
+      padding: 1rem;
+    }
+    margin: 2rem 0;
+    border-radius: 4px;
+  }
+`;
+
+const HashCode = styled(Code)`
+  word-break: break-all;
+`;
+
+const SectionTitle = styled(Heading)`
+  margin-top: 30px;
+  border-bottom: 1px solid lightgray;
+  padding-bottom: 10px;
+`;
+
+const SubSectionTitle = styled(SectionTitle)`
+  border-bottom: 0;
+`;
+
+export const Withdrawals = () => {
+  const { formatMessage } = useIntl();
+  return (
+    <PageTemplate
+      title={formatMessage({ defaultMessage: 'Staking withdrawals' })}
+      description={formatMessage({
+        defaultMessage:
+          'Information for validator operators regarding staking withdrawals.',
+      })}
+    >
+      <ComponentStyles>
+        <section>
+          <Text>
+            <FormattedMessage
+              defaultMessage="The Shanghai/Capella upgrade enabled Beacon Chain staking
+              withdrawals, with the inclusion of {eip}. This Ethereum improvement proposal
+              provides a mechanism of continuously distributing rewards to stakers, and also
+              providing a way for exited validators to unlock their entire balance—no gas
+              required."
+              values={{
+                eip: (
+                  <Link to="https://eips.ethereum.org/EIPS/eip-4895">
+                    <FormattedMessage defaultMessage="EIP-4895" />
+                  </Link>
+                ),
+              }}
+            />
+          </Text>
+
+          <section>
+            <Anchor to="#enabling-withdrawals" id="enabling-withdrawals">
+              <SectionTitle level={2} className="mb10">
+                <FormattedMessage defaultMessage="Enabling withdrawals" />
+              </SectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="For your validator account to be capable of withdrawals of any
+                kind, a withdrawal address must be provided and registered to your account on the
+                Beacon Chain. This should be an address you control, and cannot be changed once set."
+              />
+            </Text>
+
+            <WithdrawalsTabComparison />
+          </section>
+
+          <section>
+            <Anchor to="#update-your-keys" id="update-your-keys">
+              <SubSectionTitle level={3} className="mb10">
+                <FormattedMessage defaultMessage="How to update validator keys" />
+              </SubSectionTitle>
+            </Anchor>
+            <Text className="mb10">
+              <FormattedMessage
+                defaultMessage="If your validator account still has BLS withdrawal credentials ({type0}),
```

Comment:
This is the first time BLS credentials are mentioned. Could we add a link to an external resource that explains what this is or what this represents? or are we assuming the reader already knows this?

Reply:
I wouldn't _assume_ they know about this, but also not sure we need to link out and distract. I think maybe a one-sentence explainer that explains.

Would there be benefit to laying something like this out?
- `0x00` = "Type 0" = BLS keys = old withdrawal credentials -> not eligible for withdrawals
- `0x01` = "Type 1" = Execution keys = new withdrawal credentials -> withdrawals enabled

Reply:
I like that! I think it clarifies a lot, at least it did to me.

Note: not a blocker, we could add that on a separate PR if you prefer.

Reply:
We can add it here... This branch has been merged into `shapella` for the Zhejiang version of the page, but is the working branch for the whole withdrawals addition still. 

The intent was to test on Zhejiang and leave this open to iterate more if/as needed, then merge into `master` (Goerli) and `mainnet` (Mainnet) when ready.

Reply:
<img width="803" alt="image" src="https://user-images.githubusercontent.com/54227730/217416209-da5ae7b6-9e29-422c-9b57-f55267fdf2a2.png">
cc: @konopkja... Maybe something simple like this?

Reply:
would be good to make it more visually clear like having a red cross emoji for type 0 and green checkmark for type 1